lib/spring/test/acceptance_test.rb

```diff
@@ -47,6 +47,14 @@ def assert_failure(command, expected_output = nil)
         assert_output artifacts, expected_output if expected_output
       end
 
+      def assert_not_include(command, not_expected)
```

Comment:
let's try to not override them :D
maybe something different like refute_output_includes or so ?

On Sat, Dec 12, 2015 at 11:40 PM, Aditya Prakash <notifications@github.com>
wrote:

> In lib/spring/test/acceptance_test.rb
> <https://github.com/rails/spring/pull/455#discussion_r47441407>:
>
> > @@ -47,6 +47,14 @@ def assert_failure(command, expected_output = nil)
> >          assert_output artifacts, expected_output if expected_output
> >        end
> >
> > +      def assert_not_include(command, not_expected)
>
> assert_not_include is defined in test::unit
> <http://www.rubydoc.info/github/test-unit/test-unit/Test%2FUnit%2FAssertions%3Aassert_not_include>
> .
>
> assert_output defined in minitest
> <http://ruby-doc.org/stdlib-1.9.3/libdoc/minitest/unit/rdoc/MiniTest/Assertions.html#method-i-assert_output>
> is also overloaded here. Should I change assert_not_include to
> refute_include anyway?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/spring/pull/455/files#r47441407>.
>